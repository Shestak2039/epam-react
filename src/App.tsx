import React from 'react';

import Header from './components/Header/Header';
import Wrapper from './components/Wrapper/Wrapper';
import Main from './components/Main/Main';
import SortingFilms from './components/SortingFilms/SortingFilms';

const App: React.FunctionComponent = () => (
    <Wrapper>
        <Header />
        <Main>
            <SortingFilms />
        </Main>
    </Wrapper>
);

export default App;
