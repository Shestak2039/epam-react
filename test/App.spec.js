import React from 'react';
import renderer from 'react-test-renderer';
 
import App from '../src/App';
 
describe('Counter', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});