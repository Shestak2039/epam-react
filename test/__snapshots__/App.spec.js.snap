// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
<h1>
  Hello world
</h1>
`;
